plugins/optimization-detective/optimization.php

```diff
@@ -10,6 +10,8 @@
 	exit; // Exit if accessed directly.
 }
 
+const OD_DETECTION_SCRIPT_PLACEHOLDER_COMMENT = '<!--OPTIMIZATION_DETECTIVE_DETECTION_SCRIPT-->';
```

Comment:
This is probably safer, since another optimization plugin may strip out comments.

```suggestion
const OD_DETECTION_SCRIPT_PLACEHOLDER_COMMENT = '{{{OPTIMIZATION_DETECTIVE_DETECTION_SCRIPT}}}';
```